### PR TITLE
adding new file on Feature branch

### PR DESCRIPTION
**Pull request by KareemZafan**

### CODE REVIEW

File: newFile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+New file on Feature1 branch
```
Comment (ghost) [inline newFile.txt:1]: need to be improved.

Comment (Abdelrahmanbeshiir) [inline newFile.txt:1]: need to be improved

**Review by ghost — DISMISSED** [review]

File: newFile.txt (hunk 1 above)
Comment (Rana29hesham) [inline newFile.txt:1]: good

**Review by Rana29hesham — APPROVED** [review]

**Review by Abdelrahmanbeshiir — APPROVED** [review]

File: newFile.txt (hunk 1 above)
Comment (KareemZafan) [reply newFile.txt:1]: Okay i will enhance

**Pull request merged**